tests/cmd/numerical.txt

```diff
@@ -1,2 +1,2 @@
 brun -n '(q (1 2 3))'
```

Comment:
Maybe we change this one to `(q . (1 2 3))` instead? I don't remember what the intent of this test is supposed to be.